lib/task-data/tasks/secure-erase-drive.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     injectableName: 'Task.Drive.SecureErase',
     implementsTask: 'Task.Base.SecureErase',
     options: {
-        eraseSettings: []
+        eraseSettings: [],
+        baseUri: '{{ file.server }}'
```

Comment:
In the case where external file server is used, file.server will be the external file server address and the notification will not be sent to correct server. Replace this with api.server sounds more corect. @iceiilin